src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -70,30 +70,41 @@ const SuggestedOwners = React.createClass({
     });
   },
 
+  assignTo(member) {
+    this.api.assignTo({id: this.props.event.groupID, member: member});
+  },
+
   renderCommitter({author, commits}) {
     return (
-      <span key={author.id} className="avatar-grid-item tip" title={
+      <span key={author.id} className="avatar-grid-item tip" onClick={()=>this.assignTo(author)} title={
```

Comment:
i usually put spaces like `() => this.assignTo..` but looks like we already have a few cases of spacing like this so whatevs